evm/src/generation/state.rs

```diff
@@ -19,9 +19,6 @@ pub(crate) struct GenerationState<F: Field> {
 
     pub(crate) keccak_inputs: Vec<[u64; keccak::keccak_stark::NUM_INPUTS]>,
     pub(crate) logic_ops: Vec<logic::Operation>,
-
-    /// Non-deterministic inputs provided by the prover.
-    pub(crate) prover_inputs: Vec<U256>,
```

Comment:
(This single prover input buffer no longer really makes sense, now that William's `PROVER_INPUT` change lets us tell the prover what kind of input is requested. We might add more specific buffers for e.g. transaction data later though.)